decision_maker.go

```diff
@@ -212,12 +216,14 @@ func inspectUpgradeScenario(r *release, rs releaseState) {
 
 	if r.Namespace == rs.Namespace {
 
-		version, msg := getChartVersion(r)
-		if msg != "" {
-			logError(msg)
-			return
+		if !isLocalChart(r.Chart) {
```

Comment:
This part of the patch should go into another PR and be merged as soon as possible.

Reply:
I have removed this patch from this PR. Awaiting PR #291 instead.

Reply:
I have removed this patch from this PR. Awaiting PR #291 instead.

Reply:
I have removed this patch from this PR. Awaiting PR #291 instead.